<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 3DGM - Analysis | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast</title>
  <meta name="description" content="Gahagan biface shape change." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 3DGM - Analysis | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Gahagan biface shape change." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 3DGM - Analysis | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast" />
  
  <meta name="twitter:description" content="Gahagan biface shape change." />
  

<meta name="author" content="Robert Z. Selden, Jr., John E. Dockall, and David L. Carlson" />


<meta name="date" content="2021-12-17" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="dgm---landmarking-protocol.html"/>
<link rel="next" href="references.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">gahaganmorph.3</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#basis-of-inquiry"><i class="fa fa-check"></i><b>0.1</b> Basis of inquiry</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#primary-findings"><i class="fa fa-check"></i><b>0.2</b> Primary findings</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="index.html"><a href="index.html#increase-in-mortuary-assemblage-diversity-as-a-function-of-time"><i class="fa fa-check"></i><b>0.2.1</b> Increase in mortuary assemblage diversity as a function of time</a></li>
<li class="chapter" data-level="0.2.2" data-path="index.html"><a href="index.html#shape-preference-as-a-function-of-burial-practice"><i class="fa fa-check"></i><b>0.2.2</b> Shape preference as a function of burial practice</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i>Data management</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i>Colophon</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="calibrate-model-legacy-dates.html"><a href="calibrate-model-legacy-dates.html"><i class="fa fa-check"></i><b>1</b> Calibrate + Model Legacy Dates</a>
<ul>
<li class="chapter" data-level="1.1" data-path="calibrate-model-legacy-dates.html"><a href="calibrate-model-legacy-dates.html#calibration-and-plots-for-dates-from-burial-pit-5-at-the-mounds-plantation-site"><i class="fa fa-check"></i><b>1.1</b> Calibration and plots for dates from Burial Pit 5 at the Mounds Plantation site</a></li>
<li class="chapter" data-level="1.2" data-path="calibrate-model-legacy-dates.html"><a href="calibrate-model-legacy-dates.html#calibration-and-plots-for-dates-from-burial-pit-2-at-the-gahagan-mound-site"><i class="fa fa-check"></i><b>1.2</b> Calibration and plots for dates from Burial Pit 2 at the Gahagan Mound site</a></li>
<li class="chapter" data-level="1.3" data-path="calibrate-model-legacy-dates.html"><a href="calibrate-model-legacy-dates.html#calibration-and-plots-for-the-dates-from-f119-and-f134-at-the-george-c.-davis-site"><i class="fa fa-check"></i><b>1.3</b> Calibration and plots for the dates from F119 and F134 at the George C. Davis site</a></li>
<li class="chapter" data-level="1.4" data-path="calibrate-model-legacy-dates.html"><a href="calibrate-model-legacy-dates.html#chronological-model-for-burial-contexts-yielding-gahagan-bifaces"><i class="fa fa-check"></i><b>1.4</b> Chronological model for burial contexts yielding Gahagan bifaces</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="seriation.html"><a href="seriation.html"><i class="fa fa-check"></i><b>2</b> Seriation</a></li>
<li class="chapter" data-level="3" data-path="composition-diversity.html"><a href="composition-diversity.html"><i class="fa fa-check"></i><b>3</b> Composition &amp; Diversity</a>
<ul>
<li class="chapter" data-level="3.1" data-path="composition-diversity.html"><a href="composition-diversity.html#taxonomic-composition"><i class="fa fa-check"></i><b>3.1</b> Taxonomic composition</a></li>
<li class="chapter" data-level="3.2" data-path="composition-diversity.html"><a href="composition-diversity.html#alpha-diversity"><i class="fa fa-check"></i><b>3.2</b> Alpha diversity</a></li>
<li class="chapter" data-level="3.3" data-path="composition-diversity.html"><a href="composition-diversity.html#relative-abundance"><i class="fa fa-check"></i><b>3.3</b> Relative abundance</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="composition-diversity.html"><a href="composition-diversity.html#relative-abundance-by-context"><i class="fa fa-check"></i><b>3.3.1</b> Relative abundance by context</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="composition-diversity.html"><a href="composition-diversity.html#diversity-indices"><i class="fa fa-check"></i><b>3.4</b> Diversity indices</a></li>
<li class="chapter" data-level="3.5" data-path="composition-diversity.html"><a href="composition-diversity.html#beta-diversity"><i class="fa fa-check"></i><b>3.5</b> Beta diversity</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="composition-diversity.html"><a href="composition-diversity.html#diversity-through-time"><i class="fa fa-check"></i><b>3.5.1</b> Diversity through time</a></li>
<li class="chapter" data-level="3.5.2" data-path="composition-diversity.html"><a href="composition-diversity.html#evenness-through-time"><i class="fa fa-check"></i><b>3.5.2</b> Evenness through time</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html"><i class="fa fa-check"></i><b>4</b> 3DGM - Modeling Geometry</a>
<ul>
<li class="chapter" data-level="4.1" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html#section"><i class="fa fa-check"></i><b>4.1</b> 4078-8</a></li>
<li class="chapter" data-level="4.2" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html#section-1"><i class="fa fa-check"></i><b>4.2</b> 4078-72</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html"><i class="fa fa-check"></i><b>5</b> 3DGM - Landmarking Protocol</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#generating-the-spline"><i class="fa fa-check"></i><b>5.0.1</b> Generating the spline</a></li>
<li class="chapter" data-level="5.0.2" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#splitting-the-spline"><i class="fa fa-check"></i><b>5.0.2</b> Splitting the spline</a></li>
<li class="chapter" data-level="5.0.3" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#landmark-and-semilandmark-placement"><i class="fa fa-check"></i><b>5.0.3</b> Landmark and semilandmark placement</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="dgm---analysis.html"><a href="dgm---analysis.html"><i class="fa fa-check"></i><b>6</b> 3DGM - Analysis</a>
<ul>
<li class="chapter" data-level="6.1" data-path="dgm---analysis.html"><a href="dgm---analysis.html#load-packages-data"><i class="fa fa-check"></i><b>6.1</b> Load packages + data</a></li>
<li class="chapter" data-level="6.2" data-path="dgm---analysis.html"><a href="dgm---analysis.html#generalized-procrustes-analysis"><i class="fa fa-check"></i><b>6.2</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="6.3" data-path="dgm---analysis.html"><a href="dgm---analysis.html#boxplot"><i class="fa fa-check"></i><b>6.3</b> Boxplot</a></li>
<li class="chapter" data-level="6.4" data-path="dgm---analysis.html"><a href="dgm---analysis.html#principal-components-analysis"><i class="fa fa-check"></i><b>6.4</b> Principal Components Analysis</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="dgm---analysis.html"><a href="dgm---analysis.html#plot-pca-with-mean-shape-xy-maximaminima"><i class="fa fa-check"></i><b>6.4.1</b> Plot PCA with mean shape + x/y maxima/minima</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="dgm---analysis.html"><a href="dgm---analysis.html#procrustes-anova"><i class="fa fa-check"></i><b>6.5</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="6.6" data-path="dgm---analysis.html"><a href="dgm---analysis.html#morphological-disparity"><i class="fa fa-check"></i><b>6.6</b> Morphological disparity</a></li>
<li class="chapter" data-level="6.7" data-path="dgm---analysis.html"><a href="dgm---analysis.html#mean-shapes"><i class="fa fa-check"></i><b>6.7</b> Mean shapes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dgm---analysis" class="section level1" number="6">
<h1><span class="header-section-number">Chapter 6</span> 3DGM - Analysis</h1>
<p>This 3D geometric morphometric analysis follows recent analyses of Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>)</span>, where Gahagan bifaces were found to differ in shape across the same geography as Caddo bottles <span class="citation">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span> and Perdiz arrow points (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3/perdiz3.pdf">Selden and Dockall, this volume</a>). The analysis, and the two that precede it, builds upon the work of Shafer <span class="citation">(<a href="#ref-RN3684" role="doc-biblioref">1973</a>, <a href="#ref-RN20701" role="doc-biblioref">1974</a>, <a href="#ref-RN4924" role="doc-biblioref">2006</a>)</span>. A succinct overview of the analytical procedures used for this analysis is provided in the manuscript, and the analytical code provided in this document can be used to reproduce the results exactly.</p>
<p><em><strong>Hypothesis: Gahagan bifaces included in Caddo burials as a cache differ in morphology from those placed atop or alongside an individual.</strong></em></p>
<p>This hypothesis is tested using Gahagan bifaces recovered from two discrete contexts; one interred as part of a <em>cache</em>, and the other placed atop or alongside <em>individuals</em>. Distinct Caddo burial practices may have constrained local morphological requirements, highlighting differential <code>shape preference</code>.</p>
<div id="load-packages-data" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> Load packages + data</h2>
<div class="sourceCode" id="cb150"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb150-1"><a href="dgm---analysis.html#cb150-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load libraries</span></span>
<span id="cb150-2"><a href="dgm---analysis.html#cb150-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb150-3"><a href="dgm---analysis.html#cb150-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(geomorph)</span>
<span id="cb150-4"><a href="dgm---analysis.html#cb150-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb150-5"><a href="dgm---analysis.html#cb150-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(wesanderson)</span>
<span id="cb150-6"><a href="dgm---analysis.html#cb150-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb150-7"><a href="dgm---analysis.html#cb150-7" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&#39;readmulti.csv.R&#39;</span>)</span>
<span id="cb150-8"><a href="dgm---analysis.html#cb150-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb150-9"><a href="dgm---analysis.html#cb150-9" aria-hidden="true" tabindex="-1"></a><span class="co"># read .csv files</span></span>
<span id="cb150-10"><a href="dgm---analysis.html#cb150-10" aria-hidden="true" tabindex="-1"></a><span class="fu">setwd</span>(<span class="st">&quot;./data&quot;</span>)</span>
<span id="cb150-11"><a href="dgm---analysis.html#cb150-11" aria-hidden="true" tabindex="-1"></a>filelist <span class="ot">&lt;-</span> <span class="fu">list.files</span>(<span class="at">pattern =</span> <span class="st">&quot;.csv&quot;</span>)</span>
<span id="cb150-12"><a href="dgm---analysis.html#cb150-12" aria-hidden="true" tabindex="-1"></a>coords <span class="ot">&lt;-</span> <span class="fu">readmulti.csv</span>(filelist)</span>
<span id="cb150-13"><a href="dgm---analysis.html#cb150-13" aria-hidden="true" tabindex="-1"></a><span class="fu">setwd</span>(<span class="st">&quot;../&quot;</span>)</span>
<span id="cb150-14"><a href="dgm---analysis.html#cb150-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb150-15"><a href="dgm---analysis.html#cb150-15" aria-hidden="true" tabindex="-1"></a><span class="co"># read qualitative data</span></span>
<span id="cb150-16"><a href="dgm---analysis.html#cb150-16" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;qdata.csv&quot;</span>,<span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">row.names =</span> <span class="dv">1</span>)</span>
<span id="cb150-17"><a href="dgm---analysis.html#cb150-17" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> qdata[<span class="fu">match</span>(<span class="fu">dimnames</span>(coords)[[<span class="dv">3</span>]], <span class="fu">rownames</span>(qdata)),]</span></code></pre></div>
</div>
<div id="generalized-procrustes-analysis" class="section level2" number="6.2">
<h2><span class="header-section-number">6.2</span> Generalized Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.1.0 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2021</a>)</span> using the <code>geomorph</code> library v. 4.0.0 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otárola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<div class="sourceCode" id="cb151"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb151-1"><a href="dgm---analysis.html#cb151-1" aria-hidden="true" tabindex="-1"></a>Y.gpa <span class="ot">&lt;-</span> <span class="fu">gpagen</span>(coords, </span>
<span id="cb151-2"><a href="dgm---analysis.html#cb151-2" aria-hidden="true" tabindex="-1"></a>              <span class="at">PrinAxes =</span> <span class="cn">TRUE</span>, </span>
<span id="cb151-3"><a href="dgm---analysis.html#cb151-3" aria-hidden="true" tabindex="-1"></a>              <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb151-4"><a href="dgm---analysis.html#cb151-4" aria-hidden="true" tabindex="-1"></a><span class="do">## plot gpa</span></span>
<span id="cb151-5"><a href="dgm---analysis.html#cb151-5" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Y.gpa)</span>
<span id="cb151-6"><a href="dgm---analysis.html#cb151-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb151-7"><a href="dgm---analysis.html#cb151-7" aria-hidden="true" tabindex="-1"></a><span class="do">## 3D gpa plot</span></span>
<span id="cb151-8"><a href="dgm---analysis.html#cb151-8" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/gpa3d.png&#39;</span>)</span></code></pre></div>
<p><img src="images/gpa3d.png" width="100%" /></p>
<div class="sourceCode" id="cb152"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb152-1"><a href="dgm---analysis.html#cb152-1" aria-hidden="true" tabindex="-1"></a><span class="co"># geomorph data frame</span></span>
<span id="cb152-2"><a href="dgm---analysis.html#cb152-2" aria-hidden="true" tabindex="-1"></a>gdf <span class="ot">&lt;-</span> <span class="fu">geomorph.data.frame</span>(<span class="at">shape =</span> Y.gpa<span class="sc">$</span>coords, </span>
<span id="cb152-3"><a href="dgm---analysis.html#cb152-3" aria-hidden="true" tabindex="-1"></a>                         <span class="at">size =</span> Y.gpa<span class="sc">$</span>Csize,</span>
<span id="cb152-4"><a href="dgm---analysis.html#cb152-4" aria-hidden="true" tabindex="-1"></a>                         <span class="at">context =</span> qdata<span class="sc">$</span>context) </span>
<span id="cb152-5"><a href="dgm---analysis.html#cb152-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb152-6"><a href="dgm---analysis.html#cb152-6" aria-hidden="true" tabindex="-1"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb152-7"><a href="dgm---analysis.html#cb152-7" aria-hidden="true" tabindex="-1"></a>qdata<span class="sc">$</span>csz <span class="ot">&lt;-</span> Y.gpa<span class="sc">$</span>Csize</span>
<span id="cb152-8"><a href="dgm---analysis.html#cb152-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb152-9"><a href="dgm---analysis.html#cb152-9" aria-hidden="true" tabindex="-1"></a><span class="co"># attributes for boxplot</span></span>
<span id="cb152-10"><a href="dgm---analysis.html#cb152-10" aria-hidden="true" tabindex="-1"></a>csz <span class="ot">&lt;-</span> qdata<span class="sc">$</span>csz</span>
<span id="cb152-11"><a href="dgm---analysis.html#cb152-11" aria-hidden="true" tabindex="-1"></a>context <span class="ot">&lt;-</span> qdata<span class="sc">$</span>context</span>
<span id="cb152-12"><a href="dgm---analysis.html#cb152-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb152-13"><a href="dgm---analysis.html#cb152-13" aria-hidden="true" tabindex="-1"></a><span class="co"># palette</span></span>
<span id="cb152-14"><a href="dgm---analysis.html#cb152-14" aria-hidden="true" tabindex="-1"></a>pal <span class="ot">&lt;-</span> <span class="fu">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)</span></code></pre></div>
</div>
<div id="boxplot" class="section level2" number="6.3">
<h2><span class="header-section-number">6.3</span> Boxplot</h2>
<div class="sourceCode" id="cb153"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb153-1"><a href="dgm---analysis.html#cb153-1" aria-hidden="true" tabindex="-1"></a><span class="co"># boxplot - centroid size by context</span></span>
<span id="cb153-2"><a href="dgm---analysis.html#cb153-2" aria-hidden="true" tabindex="-1"></a>csz.temp <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(qdata, <span class="fu">aes</span>(<span class="at">x =</span> context, <span class="at">y =</span> csz, <span class="at">color =</span> context)) <span class="sc">+</span></span>
<span id="cb153-3"><a href="dgm---analysis.html#cb153-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb153-4"><a href="dgm---analysis.html#cb153-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_dotplot</span>(<span class="at">binaxis =</span> <span class="st">&#39;y&#39;</span>, <span class="at">stackdir =</span> <span class="st">&#39;center&#39;</span>, <span class="at">dotsize =</span> <span class="fl">0.3</span>) <span class="sc">+</span></span>
<span id="cb153-5"><a href="dgm---analysis.html#cb153-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> pal) <span class="sc">+</span></span>
<span id="cb153-6"><a href="dgm---analysis.html#cb153-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span></span>
<span id="cb153-7"><a href="dgm---analysis.html#cb153-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&#39;Context&#39;</span>, <span class="at">y =</span> <span class="st">&#39;Centroid Size&#39;</span>)</span>
<span id="cb153-8"><a href="dgm---analysis.html#cb153-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb153-9"><a href="dgm---analysis.html#cb153-9" aria-hidden="true" tabindex="-1"></a><span class="do">## render plot</span></span>
<span id="cb153-10"><a href="dgm---analysis.html#cb153-10" aria-hidden="true" tabindex="-1"></a>csz.temp</span></code></pre></div>
<pre><code>## Bin width defaults to 1/30 of the range of the data. Pick better value with `binwidth`.</code></pre>
<p><img src="_main_files/figure-html/centsize.box.h2-1.png" width="100%" /></p>
</div>
<div id="principal-components-analysis" class="section level2" number="6.4">
<h2><span class="header-section-number">6.4</span> Principal Components Analysis</h2>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb155"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb155-1"><a href="dgm---analysis.html#cb155-1" aria-hidden="true" tabindex="-1"></a><span class="co"># principal components analysis</span></span>
<span id="cb155-2"><a href="dgm---analysis.html#cb155-2" aria-hidden="true" tabindex="-1"></a>pca <span class="ot">&lt;-</span> <span class="fu">gm.prcomp</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb155-3"><a href="dgm---analysis.html#cb155-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 66 
## Number of vectors 65 
## 
## Importance of Components:
##                             Comp1        Comp2        Comp3        Comp4
## Eigenvalues            0.00354545 0.0004879563 0.0002164074 0.0001321895
## Proportion of Variance 0.72578649 0.0998891644 0.0443005890 0.0270604221
## Cumulative Proportion  0.72578649 0.8256756552 0.8699762442 0.8970366663
##                               Comp5        Comp6        Comp7        Comp8
## Eigenvalues            0.0001143175 8.404064e-05 6.702847e-05 5.067751e-05
## Proportion of Variance 0.0234018531 1.720390e-02 1.372135e-02 1.037415e-02
## Cumulative Proportion  0.9204385194 9.376424e-01 9.513638e-01 9.617379e-01
##                               Comp9       Comp10       Comp11       Comp12
## Eigenvalues            0.0000440233 1.898037e-05 1.736646e-05 1.284811e-05
## Proportion of Variance 0.0090119773 3.885457e-03 3.555074e-03 2.630127e-03
## Cumulative Proportion  0.9707498933 9.746354e-01 9.781904e-01 9.808206e-01
##                              Comp13       Comp14       Comp15       Comp16
## Eigenvalues            1.100218e-05 8.895681e-06 7.742354e-06 7.187114e-06
## Proportion of Variance 2.252248e-03 1.821028e-03 1.584931e-03 1.471269e-03
## Cumulative Proportion  9.830728e-01 9.848938e-01 9.864788e-01 9.879500e-01
##                              Comp17       Comp18       Comp19       Comp20
## Eigenvalues            5.664377e-06 5.004952e-06 4.648354e-06 3.900183e-06
## Proportion of Variance 1.159550e-03 1.024560e-03 9.515611e-04 7.984035e-04
## Cumulative Proportion  9.891096e-01 9.901341e-01 9.910857e-01 9.918841e-01
##                              Comp21       Comp22       Comp23       Comp24
## Eigenvalues            3.195324e-06 2.996739e-06 2.844847e-06 2.666197e-06
## Proportion of Variance 6.541124e-04 6.134602e-04 5.823664e-04 5.457951e-04
## Cumulative Proportion  9.925382e-01 9.931517e-01 9.937340e-01 9.942798e-01
##                              Comp25       Comp26       Comp27       Comp28
## Eigenvalues            2.337022e-06 2.170575e-06 1.931296e-06 1.785836e-06
## Proportion of Variance 4.784100e-04 4.443368e-04 3.953541e-04 3.655772e-04
## Cumulative Proportion  9.947582e-01 9.952026e-01 9.955979e-01 9.959635e-01
##                              Comp29       Comp30       Comp31       Comp32
## Eigenvalues            1.586191e-06 1.466620e-06 1.443472e-06 1.200869e-06
## Proportion of Variance 3.247079e-04 3.002307e-04 2.954920e-04 2.458291e-04
## Cumulative Proportion  9.962882e-01 9.965885e-01 9.968839e-01 9.971298e-01
##                              Comp33       Comp34       Comp35       Comp36
## Eigenvalues            1.134647e-06 1.048795e-06 1.002741e-06 9.456084e-07
## Proportion of Variance 2.322727e-04 2.146980e-04 2.052703e-04 1.935748e-04
## Cumulative Proportion  9.973620e-01 9.975767e-01 9.977820e-01 9.979756e-01
##                              Comp37       Comp38       Comp39       Comp40
## Eigenvalues            8.204437e-07 7.873510e-07 6.408918e-07 6.223865e-07
## Proportion of Variance 1.679524e-04 1.611780e-04 1.311965e-04 1.274083e-04
## Cumulative Proportion  9.981435e-01 9.983047e-01 9.984359e-01 9.985633e-01
##                              Comp41       Comp42       Comp43       Comp44
## Eigenvalues            5.776848e-07 5.189095e-07 5.059827e-07 4.682170e-07
## Proportion of Variance 1.182574e-04 1.062256e-04 1.035793e-04 9.584836e-05
## Cumulative Proportion  9.986816e-01 9.987878e-01 9.988914e-01 9.989872e-01
##                              Comp45       Comp46       Comp47       Comp48
## Eigenvalues            4.472489e-07 4.240362e-07 4.050463e-07 3.713485e-07
## Proportion of Variance 9.155599e-05 8.680412e-05 8.291672e-05 7.601848e-05
## Cumulative Proportion  9.990788e-01 9.991656e-01 9.992485e-01 9.993245e-01
##                              Comp49       Comp50       Comp51       Comp52
## Eigenvalues            3.532916e-07 3.212258e-07 3.099419e-07 2.623062e-07
## Proportion of Variance 7.232206e-05 6.575789e-05 6.344797e-05 5.369651e-05
## Cumulative Proportion  9.993969e-01 9.994626e-01 9.995261e-01 9.995798e-01
##                              Comp53       Comp54       Comp55       Comp56
## Eigenvalues            2.582888e-07 2.349982e-07 2.256611e-07 2.057828e-07
## Proportion of Variance 5.287411e-05 4.810630e-05 4.619492e-05 4.212564e-05
## Cumulative Proportion  9.996326e-01 9.996807e-01 9.997269e-01 9.997691e-01
##                              Comp57       Comp58       Comp59       Comp60
## Eigenvalues            1.806898e-07 1.667148e-07 1.562736e-07 1.462683e-07
## Proportion of Variance 3.698888e-05 3.412806e-05 3.199065e-05 2.994248e-05
## Cumulative Proportion  9.998060e-01 9.998402e-01 9.998722e-01 9.999021e-01
##                              Comp61       Comp62       Comp63      Comp64
## Eigenvalues            1.314886e-07 1.072852e-07 8.676582e-08 8.44168e-08
## Proportion of Variance 2.691693e-05 2.196227e-05 1.776176e-05 1.72809e-05
## Cumulative Proportion  9.999290e-01 9.999510e-01 9.999687e-01 9.99986e-01
##                              Comp65
## Eigenvalues            6.824529e-08
## Proportion of Variance 1.397044e-05
## Cumulative Proportion  1.000000e+00</code></pre>
<div class="sourceCode" id="cb157"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb157-1"><a href="dgm---analysis.html#cb157-1" aria-hidden="true" tabindex="-1"></a><span class="co"># set plot parameters to plot by context</span></span>
<span id="cb157-2"><a href="dgm---analysis.html#cb157-2" aria-hidden="true" tabindex="-1"></a>pch.gps.context <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">15</span>,<span class="dv">17</span>)[<span class="fu">as.factor</span>(context)]</span>
<span id="cb157-3"><a href="dgm---analysis.html#cb157-3" aria-hidden="true" tabindex="-1"></a>col.gps.context <span class="ot">&lt;-</span> pal[<span class="fu">as.factor</span>(context)]</span>
<span id="cb157-4"><a href="dgm---analysis.html#cb157-4" aria-hidden="true" tabindex="-1"></a>col.hull.context <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;#C27D38&quot;</span>,<span class="st">&quot;#798E87&quot;</span>)</span>
<span id="cb157-5"><a href="dgm---analysis.html#cb157-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb157-6"><a href="dgm---analysis.html#cb157-6" aria-hidden="true" tabindex="-1"></a><span class="do">## plot pca by context 2</span></span>
<span id="cb157-7"><a href="dgm---analysis.html#cb157-7" aria-hidden="true" tabindex="-1"></a>pc.plot <span class="ot">&lt;-</span> <span class="fu">plot</span>(pca, <span class="at">asp =</span> <span class="dv">1</span>,</span>
<span id="cb157-8"><a href="dgm---analysis.html#cb157-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">pch =</span> pch.gps.context,</span>
<span id="cb157-9"><a href="dgm---analysis.html#cb157-9" aria-hidden="true" tabindex="-1"></a>                  <span class="at">col =</span> col.gps.context)</span>
<span id="cb157-10"><a href="dgm---analysis.html#cb157-10" aria-hidden="true" tabindex="-1"></a>                  <span class="fu">shapeHulls</span>(pc.plot, </span>
<span id="cb157-11"><a href="dgm---analysis.html#cb157-11" aria-hidden="true" tabindex="-1"></a>                             <span class="at">groups =</span> context,</span>
<span id="cb157-12"><a href="dgm---analysis.html#cb157-12" aria-hidden="true" tabindex="-1"></a>                             <span class="at">group.cols =</span> col.hull.context)</span></code></pre></div>
<p><img src="_main_files/figure-html/pca1.h2-1.png" width="100%" /></p>
<div id="plot-pca-with-mean-shape-xy-maximaminima" class="section level3" number="6.4.1">
<h3><span class="header-section-number">6.4.1</span> Plot PCA with mean shape + x/y maxima/minima</h3>
<div class="sourceCode" id="cb158"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb158-1"><a href="dgm---analysis.html#cb158-1" aria-hidden="true" tabindex="-1"></a><span class="do">## plot PCA with mean shape (gray) and x/y maxima/minima (black)</span></span>
<span id="cb158-2"><a href="dgm---analysis.html#cb158-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/cacheind-pca-warpref.png&#39;</span>)</span></code></pre></div>
<p><img src="images/cacheind-pca-warpref.png" width="100%" /></p>
</div>
</div>
<div id="procrustes-anova" class="section level2" number="6.5">
<h2><span class="header-section-number">6.5</span> Procrustes ANOVA</h2>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Michael L. Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>. To assess whether shape changes with size (allometry), and differs by context (context), Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN1756" role="doc-biblioref">M. L. Collyer, Sekora, and Adams 2015</a>)</span>.</p>
<div class="sourceCode" id="cb159"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb159-1"><a href="dgm---analysis.html#cb159-1" aria-hidden="true" tabindex="-1"></a><span class="co"># MODEL: shape as a function of context</span></span>
<span id="cb159-2"><a href="dgm---analysis.html#cb159-2" aria-hidden="true" tabindex="-1"></a>fit.shapecontext <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> context,</span>
<span id="cb159-3"><a href="dgm---analysis.html#cb159-3" aria-hidden="true" tabindex="-1"></a>                            <span class="at">data =</span> gdf,</span>
<span id="cb159-4"><a href="dgm---analysis.html#cb159-4" aria-hidden="true" tabindex="-1"></a>                            <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb159-5"><a href="dgm---analysis.html#cb159-5" aria-hidden="true" tabindex="-1"></a>                            <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb159-6"><a href="dgm---analysis.html#cb159-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb159-7"><a href="dgm---analysis.html#cb159-7" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA: do gahagan biface shapes differ by context?</span></span>
<span id="cb159-8"><a href="dgm---analysis.html#cb159-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.shapecontext)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## context    1 0.06679 0.066785 0.21033 17.047 3.1553  1e-04 ***
## Residuals 64 0.25074 0.003918 0.78967                         
## Total     65 0.31752                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ context, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb161"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb161-1"><a href="dgm---analysis.html#cb161-1" aria-hidden="true" tabindex="-1"></a><span class="co"># MODEL: size as a function of context</span></span>
<span id="cb161-2"><a href="dgm---analysis.html#cb161-2" aria-hidden="true" tabindex="-1"></a>fit.sizecontext <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(size <span class="sc">~</span> context,</span>
<span id="cb161-3"><a href="dgm---analysis.html#cb161-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">data =</span> gdf,</span>
<span id="cb161-4"><a href="dgm---analysis.html#cb161-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb161-5"><a href="dgm---analysis.html#cb161-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb161-6"><a href="dgm---analysis.html#cb161-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb161-7"><a href="dgm---analysis.html#cb161-7" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA: do gahagan biface sizes differ by context?</span></span>
<span id="cb161-8"><a href="dgm---analysis.html#cb161-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sizecontext)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df     SS    MS     Rsq      F       Z Pr(&gt;F)
## context    1  10400 10400 0.01138 0.7365 0.32696 0.3933
## Residuals 64 903652 14120 0.98862                      
## Total     65 914052                                    
## 
## Call: procD.lm(f1 = size ~ context, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
<div id="morphological-disparity" class="section level2" number="6.6">
<h2><span class="header-section-number">6.6</span> Morphological disparity</h2>
<div class="sourceCode" id="cb163"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb163-1"><a href="dgm---analysis.html#cb163-1" aria-hidden="true" tabindex="-1"></a><span class="co"># does one context display greater **shape** variation among </span></span>
<span id="cb163-2"><a href="dgm---analysis.html#cb163-2" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb163-3"><a href="dgm---analysis.html#cb163-3" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(fit.shapecontext, </span>
<span id="cb163-4"><a href="dgm---analysis.html#cb163-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>context, </span>
<span id="cb163-5"><a href="dgm---analysis.html#cb163-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb163-6"><a href="dgm---analysis.html#cb163-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb163-7"><a href="dgm---analysis.html#cb163-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.shapecontext, groups = qdata$context,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##       cache  individual 
## 0.004027607 0.003084870 
## 
## 
## Pairwise absolute differences between variances
##                   cache   individual
## cache      0.0000000000 0.0009427367
## individual 0.0009427367 0.0000000000
## 
## 
## P-Values
##             cache individual
## cache      1.0000     0.5627
## individual 0.5627     1.0000</code></pre>
<div class="sourceCode" id="cb165"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb165-1"><a href="dgm---analysis.html#cb165-1" aria-hidden="true" tabindex="-1"></a><span class="co"># does one context display greater **size** variation among </span></span>
<span id="cb165-2"><a href="dgm---analysis.html#cb165-2" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb165-3"><a href="dgm---analysis.html#cb165-3" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(fit.sizecontext, </span>
<span id="cb165-4"><a href="dgm---analysis.html#cb165-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>context, </span>
<span id="cb165-5"><a href="dgm---analysis.html#cb165-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb165-6"><a href="dgm---analysis.html#cb165-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb165-7"><a href="dgm---analysis.html#cb165-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.sizecontext, groups = qdata$context,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##      cache individual 
##   9620.881  26412.997 
## 
## 
## Pairwise absolute differences between variances
##               cache individual
## cache          0.00   16792.12
## individual 16792.12       0.00
## 
## 
## P-Values
##            cache individual
## cache      1.000      0.015
## individual 0.015      1.000</code></pre>
</div>
<div id="mean-shapes" class="section level2" number="6.7">
<h2><span class="header-section-number">6.7</span> Mean shapes</h2>
<div class="sourceCode" id="cb167"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb167-1"><a href="dgm---analysis.html#cb167-1" aria-hidden="true" tabindex="-1"></a><span class="co"># subset landmark coordinates to produce mean shapes for contexts</span></span>
<span id="cb167-2"><a href="dgm---analysis.html#cb167-2" aria-hidden="true" tabindex="-1"></a>new.coords <span class="ot">&lt;-</span> <span class="fu">coords.subset</span>(<span class="at">A =</span> Y.gpa<span class="sc">$</span>coords,</span>
<span id="cb167-3"><a href="dgm---analysis.html#cb167-3" aria-hidden="true" tabindex="-1"></a>                            <span class="at">group =</span> qdata<span class="sc">$</span>context)</span>
<span id="cb167-4"><a href="dgm---analysis.html#cb167-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb167-5"><a href="dgm---analysis.html#cb167-5" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(new.coords)</span></code></pre></div>
<pre><code>## [1] &quot;cache&quot;      &quot;individual&quot;</code></pre>
<div class="sourceCode" id="cb169"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb169-1"><a href="dgm---analysis.html#cb169-1" aria-hidden="true" tabindex="-1"></a><span class="do">## plot shape means</span></span>
<span id="cb169-2"><a href="dgm---analysis.html#cb169-2" aria-hidden="true" tabindex="-1"></a>mean <span class="ot">&lt;-</span> <span class="fu">lapply</span>(new.coords, mshape)</span>
<span id="cb169-3"><a href="dgm---analysis.html#cb169-3" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>cache)</span>
<span id="cb169-4"><a href="dgm---analysis.html#cb169-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>individual)</span>
<span id="cb169-5"><a href="dgm---analysis.html#cb169-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb169-6"><a href="dgm---analysis.html#cb169-6" aria-hidden="true" tabindex="-1"></a><span class="co"># comparison plots</span></span>
<span id="cb169-7"><a href="dgm---analysis.html#cb169-7" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>cache,</span>
<span id="cb169-8"><a href="dgm---analysis.html#cb169-8" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>individual, </span>
<span id="cb169-9"><a href="dgm---analysis.html#cb169-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb169-10"><a href="dgm---analysis.html#cb169-10" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb169-11"><a href="dgm---analysis.html#cb169-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb169-12"><a href="dgm---analysis.html#cb169-12" aria-hidden="true" tabindex="-1"></a><span class="do">## 3D plot - final figure</span></span>
<span id="cb169-13"><a href="dgm---analysis.html#cb169-13" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/mshape-cache-ind.jpg&#39;</span>)</span></code></pre></div>
<p><img src="images/mshape-cache-ind.jpg" width="100%" /></p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN1655" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package ’geomorph’: Geometric Morphometric Analyses of 2D/3D Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otárola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN1756" class="csl-entry">
Collyer, M. L., D. J. Sekora, and D. C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2021. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN11801" class="csl-entry">
———. 2018a. <span>“<span class="nocase">A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area</span>.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN3684" class="csl-entry">
Shafer, Harry J. 1973. <span>“<span class="nocase">Lithic Technology at the George C. Davis Site, Cherokee County, Texas</span>.”</span> Unpublished Ph.D. Dissertation, The University of Texas at Austin.
</div>
<div id="ref-RN20701" class="csl-entry">
———. 1974. <span>“<span class="nocase">Lithic Reduction Strategies at the George C. Davis Site</span>.”</span> <em>Louisiana Archaeology</em> 1: 66–74. <a href="https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf">https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf</a>.
</div>
<div id="ref-RN4924" class="csl-entry">
———. 2006. <span>“<span class="nocase">People of the Prairie: A Possible Connection to the Davis Site Caddo</span>.”</span> Texas Department of Transportation; Prewitt &amp; Associates, Inc.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, E., D. J. Gower, C. P. Klingenberg, and M. Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="dgm---landmarking-protocol.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
