<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 3DGM - Temporal | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast</title>
  <meta name="description" content="Gahagan biface shape change." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 3DGM - Temporal | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Gahagan biface shape change." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 3DGM - Temporal | Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast" />
  
  <meta name="twitter:description" content="Gahagan biface shape change." />
  

<meta name="author" content="Robert Z. Selden, Jr., John E. Dockall, and David L. Carlson" />


<meta name="date" content="2022-01-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="dgm---individual-vs.-cache.html"/>
<link rel="next" href="references.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">gahaganmorph.3</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#basis-of-inquiry"><i class="fa fa-check"></i><b>0.1</b> Basis of inquiry</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#primary-findings"><i class="fa fa-check"></i><b>0.2</b> Primary findings</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="index.html"><a href="index.html#assemblage-diversity"><i class="fa fa-check"></i><b>0.2.1</b> ASSEMBLAGE DIVERSITY</a></li>
<li class="chapter" data-level="0.2.2" data-path="index.html"><a href="index.html#object-placement"><i class="fa fa-check"></i><b>0.2.2</b> OBJECT PLACEMENT</a></li>
<li class="chapter" data-level="0.2.3" data-path="index.html"><a href="index.html#temporal-trajectories"><i class="fa fa-check"></i><b>0.2.3</b> TEMPORAL TRAJECTORIES</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i>Data management</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i>Colophon</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="recalibrate-legacy-dates.html"><a href="recalibrate-legacy-dates.html"><i class="fa fa-check"></i><b>1</b> Recalibrate Legacy Dates</a>
<ul>
<li class="chapter" data-level="1.1" data-path="recalibrate-legacy-dates.html"><a href="recalibrate-legacy-dates.html#calibration-and-plots-for-dates-from-burial-pit-5-at-the-mounds-plantation-site"><i class="fa fa-check"></i><b>1.1</b> Calibration and plots for dates from Burial Pit 5 at the Mounds Plantation site</a></li>
<li class="chapter" data-level="1.2" data-path="recalibrate-legacy-dates.html"><a href="recalibrate-legacy-dates.html#calibration-and-plots-for-dates-from-burial-pit-2-at-the-gahagan-mound-site"><i class="fa fa-check"></i><b>1.2</b> Calibration and plots for dates from Burial Pit 2 at the Gahagan Mound site</a></li>
<li class="chapter" data-level="1.3" data-path="recalibrate-legacy-dates.html"><a href="recalibrate-legacy-dates.html#calibration-and-plots-for-the-dates-from-f119-and-f134-at-the-george-c.-davis-site"><i class="fa fa-check"></i><b>1.3</b> Calibration and plots for the dates from F119 and F134 at the George C. Davis site</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="seriation.html"><a href="seriation.html"><i class="fa fa-check"></i><b>2</b> Seriation</a></li>
<li class="chapter" data-level="3" data-path="composition-diversity.html"><a href="composition-diversity.html"><i class="fa fa-check"></i><b>3</b> Composition &amp; Diversity</a>
<ul>
<li class="chapter" data-level="3.1" data-path="composition-diversity.html"><a href="composition-diversity.html#taxonomic-composition"><i class="fa fa-check"></i><b>3.1</b> Taxonomic composition</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="composition-diversity.html"><a href="composition-diversity.html#reorder-contexts-based-on-seriation-results"><i class="fa fa-check"></i><b>3.1.1</b> Reorder contexts based on seriation results</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="composition-diversity.html"><a href="composition-diversity.html#alpha-diversity"><i class="fa fa-check"></i><b>3.2</b> Alpha diversity</a></li>
<li class="chapter" data-level="3.3" data-path="composition-diversity.html"><a href="composition-diversity.html#relative-abundance"><i class="fa fa-check"></i><b>3.3</b> Relative abundance</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="composition-diversity.html"><a href="composition-diversity.html#relative-abundance-by-context"><i class="fa fa-check"></i><b>3.3.1</b> Relative abundance by context</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="composition-diversity.html"><a href="composition-diversity.html#beta-diversity"><i class="fa fa-check"></i><b>3.4</b> Beta diversity</a></li>
<li class="chapter" data-level="3.5" data-path="composition-diversity.html"><a href="composition-diversity.html#diversity-through-time"><i class="fa fa-check"></i><b>3.5</b> Diversity through time</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="composition-diversity.html"><a href="composition-diversity.html#evenness-through-time"><i class="fa fa-check"></i><b>3.5.1</b> Evenness through time</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html"><i class="fa fa-check"></i><b>4</b> 3DGM - Modeling Geometry</a>
<ul>
<li class="chapter" data-level="4.1" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html#section"><i class="fa fa-check"></i><b>4.1</b> 4078-8</a></li>
<li class="chapter" data-level="4.2" data-path="dgm---modeling-geometry.html"><a href="dgm---modeling-geometry.html#section-1"><i class="fa fa-check"></i><b>4.2</b> 4078-72</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html"><i class="fa fa-check"></i><b>5</b> 3DGM - Landmarking Protocol</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#generating-the-spline"><i class="fa fa-check"></i><b>5.0.1</b> Generating the spline</a></li>
<li class="chapter" data-level="5.0.2" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#splitting-the-spline"><i class="fa fa-check"></i><b>5.0.2</b> Splitting the spline</a></li>
<li class="chapter" data-level="5.0.3" data-path="dgm---landmarking-protocol.html"><a href="dgm---landmarking-protocol.html#landmark-and-semilandmark-placement"><i class="fa fa-check"></i><b>5.0.3</b> Landmark and semilandmark placement</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html"><i class="fa fa-check"></i><b>6</b> 3DGM - Individual vs. Cache</a>
<ul>
<li class="chapter" data-level="6.1" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#load-packages-data"><i class="fa fa-check"></i><b>6.1</b> Load packages + data</a></li>
<li class="chapter" data-level="6.2" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#generalized-procrustes-analysis"><i class="fa fa-check"></i><b>6.2</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="6.3" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#boxplot"><i class="fa fa-check"></i><b>6.3</b> Boxplot</a></li>
<li class="chapter" data-level="6.4" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#principal-components-analysis"><i class="fa fa-check"></i><b>6.4</b> Principal Components Analysis</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#plot-pca-with-mean-shape-xy-maximaminima"><i class="fa fa-check"></i><b>6.4.1</b> Plot PCA with mean shape + x/y maxima/minima</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#procrustes-anova"><i class="fa fa-check"></i><b>6.5</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="6.6" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#modularity"><i class="fa fa-check"></i><b>6.6</b> Modularity</a></li>
<li class="chapter" data-level="6.7" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#morphological-integration"><i class="fa fa-check"></i><b>6.7</b> Morphological integration</a></li>
<li class="chapter" data-level="6.8" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#morphological-disparity"><i class="fa fa-check"></i><b>6.8</b> Morphological disparity</a></li>
<li class="chapter" data-level="6.9" data-path="dgm---individual-vs.-cache.html"><a href="dgm---individual-vs.-cache.html#mean-shapes"><i class="fa fa-check"></i><b>6.9</b> Mean shapes</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="dgm---temporal.html"><a href="dgm---temporal.html"><i class="fa fa-check"></i><b>7</b> 3DGM - Temporal</a>
<ul>
<li class="chapter" data-level="7.1" data-path="dgm---temporal.html"><a href="dgm---temporal.html#load-packages-data-1"><i class="fa fa-check"></i><b>7.1</b> Load packages + data</a></li>
<li class="chapter" data-level="7.2" data-path="dgm---temporal.html"><a href="dgm---temporal.html#generalized-procrustes-analysis-1"><i class="fa fa-check"></i><b>7.2</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="7.3" data-path="dgm---temporal.html"><a href="dgm---temporal.html#boxplot-1"><i class="fa fa-check"></i><b>7.3</b> Boxplot</a></li>
<li class="chapter" data-level="7.4" data-path="dgm---temporal.html"><a href="dgm---temporal.html#principal-components-analysis-1"><i class="fa fa-check"></i><b>7.4</b> Principal Components Analysis</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="dgm---temporal.html"><a href="dgm---temporal.html#plot-pca-with-mean-shape-xy-maximaminima-1"><i class="fa fa-check"></i><b>7.4.1</b> Plot PCA with mean shape + x/y maxima/minima</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="dgm---temporal.html"><a href="dgm---temporal.html#procrustes-anova-1"><i class="fa fa-check"></i><b>7.5</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="7.6" data-path="dgm---temporal.html"><a href="dgm---temporal.html#morphological-disparity-1"><i class="fa fa-check"></i><b>7.6</b> Morphological disparity</a></li>
<li class="chapter" data-level="7.7" data-path="dgm---temporal.html"><a href="dgm---temporal.html#mean-shapes-1"><i class="fa fa-check"></i><b>7.7</b> Mean shapes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mortuary assemblage diversity, Gahagan biface morphology, and the evolution of a Caddo burial tradition in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dgm---temporal" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> 3DGM - Temporal</h1>
<p>This 3D geometric morphometric analysis follows recent analyses of Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>)</span>, where Gahagan bifaces were found to differ in shape across the same geography as Caddo bottles <span class="citation">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span> and Perdiz arrow points (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3/perdiz3.pdf">Selden and Dockall, this volume</a>). The analysis, and the two that precede it, builds upon the work of Shafer <span class="citation">(<a href="#ref-RN3684" role="doc-biblioref">1973</a>, <a href="#ref-RN20701" role="doc-biblioref">1974</a>, <a href="#ref-RN4924" role="doc-biblioref">2006</a>)</span>. A succinct overview of the analytical procedures used for this analysis is provided in the manuscript, and the analytical code provided in this document can be used to reproduce the results exactly.</p>
<p><strong><em>Hypothesis</em>: Gahagan bifaces from <em>individual</em> and <em>cache</em> contexts differ through time—<em>initial</em> vs. <em>subsequent</em> deposition—as mortuary assemblage diversity increases.</strong></p>
<p>This hypothesis is tested using Gahagan bifaces recovered from four discrete contexts; the first interred with an <em>individual</em> in the earliest (<em>initial</em>) burial context at each site, the second interred as part of a <em>cache</em> in the earliest (<em>initial</em>) burial context at each site, the third interred with an <em>individual</em> in <em>subsequent</em> burial contexts, and the fourth interred as a <em>cache</em> in <em>subsequent</em> burial contexts. Distinct Caddo burial practices may have been constrained by local morphological requirements, highlighting aspects of differential shape preference as the diversity of Caddo mortuary assemblages increased.</p>
<p>The analysis employs <em>four discrete categories</em> to assess morphological changes through time:</p>
<ul>
<li><strong>Initial Individual</strong>: from initial burial contexts that included Gahagan bifaces at each site, and interred atop or alongside an individual</li>
<li><strong>Initial Cache</strong>: from initial burial contexts that included Gahagan bifaces at each site, and interred as part of a cache along the northern wall of the burial feature</li>
<li><strong>Subsequent Individual</strong>: from subsequent burial contexts that included Gahagan bifaces at each site, and interred atop or alongside an individual</li>
<li><strong>Subsequent Cache</strong>: from subsequent burial contexts that included Gahagan bifaces at each site, and interred as part of a cache along the northern wall of the burial feature</li>
</ul>
<div id="load-packages-data-1" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Load packages + data</h2>
<div class="sourceCode" id="cb214"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb214-1"><a href="dgm---temporal.html#cb214-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load libraries</span></span>
<span id="cb214-2"><a href="dgm---temporal.html#cb214-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb214-3"><a href="dgm---temporal.html#cb214-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(geomorph)</span>
<span id="cb214-4"><a href="dgm---temporal.html#cb214-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb214-5"><a href="dgm---temporal.html#cb214-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb214-6"><a href="dgm---temporal.html#cb214-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(wesanderson)</span>
<span id="cb214-7"><a href="dgm---temporal.html#cb214-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb214-8"><a href="dgm---temporal.html#cb214-8" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&#39;readmulti.csv.R&#39;</span>)</span>
<span id="cb214-9"><a href="dgm---temporal.html#cb214-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb214-10"><a href="dgm---temporal.html#cb214-10" aria-hidden="true" tabindex="-1"></a><span class="co"># read .csv files</span></span>
<span id="cb214-11"><a href="dgm---temporal.html#cb214-11" aria-hidden="true" tabindex="-1"></a><span class="fu">setwd</span>(<span class="st">&quot;./data&quot;</span>)</span>
<span id="cb214-12"><a href="dgm---temporal.html#cb214-12" aria-hidden="true" tabindex="-1"></a>filelist <span class="ot">&lt;-</span> <span class="fu">list.files</span>(<span class="at">pattern =</span> <span class="st">&quot;.csv&quot;</span>)</span>
<span id="cb214-13"><a href="dgm---temporal.html#cb214-13" aria-hidden="true" tabindex="-1"></a>coords <span class="ot">&lt;-</span> <span class="fu">readmulti.csv</span>(filelist)</span>
<span id="cb214-14"><a href="dgm---temporal.html#cb214-14" aria-hidden="true" tabindex="-1"></a><span class="fu">setwd</span>(<span class="st">&quot;../&quot;</span>)</span>
<span id="cb214-15"><a href="dgm---temporal.html#cb214-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb214-16"><a href="dgm---temporal.html#cb214-16" aria-hidden="true" tabindex="-1"></a><span class="co"># read qualitative data</span></span>
<span id="cb214-17"><a href="dgm---temporal.html#cb214-17" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;qdata.csv&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">row.names =</span> <span class="dv">1</span>)</span>
<span id="cb214-18"><a href="dgm---temporal.html#cb214-18" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> qdata[<span class="fu">match</span>(<span class="fu">dimnames</span>(coords)[[<span class="dv">3</span>]], <span class="fu">rownames</span>(qdata)),]</span>
<span id="cb214-19"><a href="dgm---temporal.html#cb214-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb214-20"><a href="dgm---temporal.html#cb214-20" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> qdata <span class="sc">%&gt;%</span> </span>
<span id="cb214-21"><a href="dgm---temporal.html#cb214-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">unite</span>(<span class="st">&quot;merged&quot;</span>, t.context<span class="sc">:</span>context, <span class="at">remove =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
</div>
<div id="generalized-procrustes-analysis-1" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Generalized Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.1.2 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2021</a>)</span> using the <code>geomorph</code> library v. 4.0.1 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otárola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<div class="sourceCode" id="cb215"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb215-1"><a href="dgm---temporal.html#cb215-1" aria-hidden="true" tabindex="-1"></a>Y.gpa <span class="ot">&lt;-</span> <span class="fu">gpagen</span>(coords,</span>
<span id="cb215-2"><a href="dgm---temporal.html#cb215-2" aria-hidden="true" tabindex="-1"></a>                <span class="at">PrinAxes =</span> <span class="cn">TRUE</span>,</span>
<span id="cb215-3"><a href="dgm---temporal.html#cb215-3" aria-hidden="true" tabindex="-1"></a>                <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb215-4"><a href="dgm---temporal.html#cb215-4" aria-hidden="true" tabindex="-1"></a><span class="do">## plot gpa</span></span>
<span id="cb215-5"><a href="dgm---temporal.html#cb215-5" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Y.gpa)</span>
<span id="cb215-6"><a href="dgm---temporal.html#cb215-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb215-7"><a href="dgm---temporal.html#cb215-7" aria-hidden="true" tabindex="-1"></a><span class="do">## 3D gpa plot</span></span>
<span id="cb215-8"><a href="dgm---temporal.html#cb215-8" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/gpa3d.png&#39;</span>)</span></code></pre></div>
<p><img src="images/gpa3d.png" width="100%" /></p>
<div class="sourceCode" id="cb216"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb216-1"><a href="dgm---temporal.html#cb216-1" aria-hidden="true" tabindex="-1"></a><span class="co"># geomorph data frame</span></span>
<span id="cb216-2"><a href="dgm---temporal.html#cb216-2" aria-hidden="true" tabindex="-1"></a>gdf <span class="ot">&lt;-</span> <span class="fu">geomorph.data.frame</span>(<span class="at">shape =</span> Y.gpa<span class="sc">$</span>coords,</span>
<span id="cb216-3"><a href="dgm---temporal.html#cb216-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">size =</span> Y.gpa<span class="sc">$</span>Csize,</span>
<span id="cb216-4"><a href="dgm---temporal.html#cb216-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">merged =</span> qdata<span class="sc">$</span>merged) </span>
<span id="cb216-5"><a href="dgm---temporal.html#cb216-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb216-6"><a href="dgm---temporal.html#cb216-6" aria-hidden="true" tabindex="-1"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb216-7"><a href="dgm---temporal.html#cb216-7" aria-hidden="true" tabindex="-1"></a>qdata<span class="sc">$</span>csz <span class="ot">&lt;-</span> Y.gpa<span class="sc">$</span>Csize</span>
<span id="cb216-8"><a href="dgm---temporal.html#cb216-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb216-9"><a href="dgm---temporal.html#cb216-9" aria-hidden="true" tabindex="-1"></a><span class="co"># attributes for boxplot</span></span>
<span id="cb216-10"><a href="dgm---temporal.html#cb216-10" aria-hidden="true" tabindex="-1"></a>csz <span class="ot">&lt;-</span> qdata<span class="sc">$</span>csz</span>
<span id="cb216-11"><a href="dgm---temporal.html#cb216-11" aria-hidden="true" tabindex="-1"></a>merged <span class="ot">&lt;-</span> qdata<span class="sc">$</span>merged</span>
<span id="cb216-12"><a href="dgm---temporal.html#cb216-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb216-13"><a href="dgm---temporal.html#cb216-13" aria-hidden="true" tabindex="-1"></a><span class="co"># palette</span></span>
<span id="cb216-14"><a href="dgm---temporal.html#cb216-14" aria-hidden="true" tabindex="-1"></a>pal <span class="ot">&lt;-</span> <span class="fu">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>, <span class="dv">4</span>, <span class="at">type =</span> <span class="st">&quot;continuous&quot;</span>)</span></code></pre></div>
</div>
<div id="boxplot-1" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Boxplot</h2>
<div class="sourceCode" id="cb217"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb217-1"><a href="dgm---temporal.html#cb217-1" aria-hidden="true" tabindex="-1"></a><span class="co"># boxplot - centroid size by context</span></span>
<span id="cb217-2"><a href="dgm---temporal.html#cb217-2" aria-hidden="true" tabindex="-1"></a>csz.temp <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(qdata, <span class="fu">aes</span>(<span class="at">x =</span> merged, <span class="at">y =</span> csz, <span class="at">color =</span> merged)) <span class="sc">+</span></span>
<span id="cb217-3"><a href="dgm---temporal.html#cb217-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb217-4"><a href="dgm---temporal.html#cb217-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_dotplot</span>(<span class="at">binaxis =</span> <span class="st">&#39;y&#39;</span>, <span class="at">stackdir =</span> <span class="st">&#39;center&#39;</span>, <span class="at">dotsize =</span> <span class="fl">0.3</span>) <span class="sc">+</span></span>
<span id="cb217-5"><a href="dgm---temporal.html#cb217-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> pal) <span class="sc">+</span></span>
<span id="cb217-6"><a href="dgm---temporal.html#cb217-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span></span>
<span id="cb217-7"><a href="dgm---temporal.html#cb217-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">&#39;Context&#39;</span>, <span class="at">y =</span> <span class="st">&#39;Centroid Size&#39;</span>)</span>
<span id="cb217-8"><a href="dgm---temporal.html#cb217-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb217-9"><a href="dgm---temporal.html#cb217-9" aria-hidden="true" tabindex="-1"></a><span class="do">## render plot</span></span>
<span id="cb217-10"><a href="dgm---temporal.html#cb217-10" aria-hidden="true" tabindex="-1"></a>csz.temp</span></code></pre></div>
<pre><code>## Bin width defaults to 1/30 of the range of the data. Pick better value with `binwidth`.</code></pre>
<p><img src="_main_files/figure-html/centsize.box.div-1.png" width="100%" /></p>
</div>
<div id="principal-components-analysis-1" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Principal Components Analysis</h2>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb219"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb219-1"><a href="dgm---temporal.html#cb219-1" aria-hidden="true" tabindex="-1"></a><span class="co"># principal components analysis</span></span>
<span id="cb219-2"><a href="dgm---temporal.html#cb219-2" aria-hidden="true" tabindex="-1"></a>pca <span class="ot">&lt;-</span> <span class="fu">gm.prcomp</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb219-3"><a href="dgm---temporal.html#cb219-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 67 
## Number of vectors 66 
## 
## Importance of Components:
##                              Comp1        Comp2        Comp3        Comp4        Comp5        Comp6
## Eigenvalues            0.003550166 0.0004871729 0.0002149155 0.0001309349 0.0001134924 8.438744e-05
## Proportion of Variance 0.727035959 0.0997678103 0.0440123963 0.0268140702 0.0232420407 1.728164e-02
## Cumulative Proportion  0.727035959 0.8268037697 0.8708161660 0.8976302363 0.9208722770 9.381539e-01
##                               Comp7        Comp8        Comp9       Comp10       Comp11       Comp12
## Eigenvalues            6.613192e-05 5.004219e-05 4.347195e-05 1.879715e-05 1.710494e-05 1.271118e-05
## Proportion of Variance 1.354311e-02 1.024810e-02 8.902591e-03 3.849455e-03 3.502910e-03 2.603114e-03
## Cumulative Proportion  9.516970e-01 9.619451e-01 9.708477e-01 9.746972e-01 9.782001e-01 9.808032e-01
##                              Comp13       Comp14       Comp15       Comp16       Comp17       Comp18
## Eigenvalues            1.084702e-05 9.137279e-06 7.637070e-06 7.190245e-06 5.591484e-06 5.147335e-06
## Proportion of Variance 2.221354e-03 1.871217e-03 1.563990e-03 1.472485e-03 1.145076e-03 1.054119e-03
## Cumulative Proportion  9.830246e-01 9.848958e-01 9.864598e-01 9.879323e-01 9.890773e-01 9.901314e-01
##                              Comp19       Comp20       Comp21       Comp22       Comp23       Comp24
## Eigenvalues            4.677345e-06 3.848612e-06 3.213207e-06 2.972832e-06 2.866993e-06 2.628820e-06
## Proportion of Variance 9.578703e-04 7.881545e-04 6.580306e-04 6.088042e-04 5.871295e-04 5.383542e-04
## Cumulative Proportion  9.910893e-01 9.918775e-01 9.925355e-01 9.931443e-01 9.937314e-01 9.942698e-01
##                              Comp25       Comp26       Comp27       Comp28       Comp29       Comp30
## Eigenvalues            2.345589e-06 2.143701e-06 1.939816e-06 1.764678e-06 1.564872e-06 1.464723e-06
## Proportion of Variance 4.803516e-04 4.390070e-04 3.972537e-04 3.613872e-04 3.204690e-04 2.999596e-04
## Cumulative Proportion  9.947501e-01 9.951892e-01 9.955864e-01 9.959478e-01 9.962683e-01 9.965682e-01
##                              Comp31       Comp32       Comp33       Comp34       Comp35       Comp36
## Eigenvalues            1.443770e-06 1.190857e-06 1.130362e-06 1.082492e-06 9.925155e-07 9.333496e-07
## Proportion of Variance 2.956687e-04 2.438747e-04 2.314861e-04 2.216828e-04 2.032566e-04 1.911400e-04
## Cumulative Proportion  9.968639e-01 9.971078e-01 9.973392e-01 9.975609e-01 9.977642e-01 9.979553e-01
##                              Comp37       Comp38       Comp39       Comp40       Comp41       Comp42
## Eigenvalues            8.191094e-07 7.800693e-07 6.362789e-07 6.171966e-07 5.736439e-07 5.117498e-07
## Proportion of Variance 1.677448e-04 1.597499e-04 1.303031e-04 1.263953e-04 1.174761e-04 1.048009e-04
## Cumulative Proportion  9.981231e-01 9.982828e-01 9.984131e-01 9.985395e-01 9.986570e-01 9.987618e-01
##                              Comp43       Comp44       Comp45       Comp46       Comp47       Comp48
## Eigenvalues            5.019392e-07 4.641774e-07 4.429005e-07 4.315733e-07 4.143465e-07 3.720453e-07
## Proportion of Variance 1.027918e-04 9.505857e-05 9.070128e-05 8.838159e-05 8.485373e-05 7.619091e-05
## Cumulative Proportion  9.988646e-01 9.989596e-01 9.990504e-01 9.991387e-01 9.992236e-01 9.992998e-01
##                              Comp49       Comp50       Comp51       Comp52       Comp53       Comp54
## Eigenvalues            3.485554e-07 3.170829e-07 3.098429e-07 2.661885e-07 2.582105e-07 2.335530e-07
## Proportion of Variance 7.138041e-05 6.493519e-05 6.345251e-05 5.451255e-05 5.287875e-05 4.782915e-05
## Cumulative Proportion  9.993712e-01 9.994361e-01 9.994995e-01 9.995541e-01 9.996069e-01 9.996548e-01
##                              Comp55       Comp56       Comp57       Comp58       Comp59       Comp60
## Eigenvalues            2.229159e-07 2.203664e-07 2.017245e-07 1.681597e-07 1.541280e-07 1.440693e-07
## Proportion of Variance 4.565079e-05 4.512867e-05 4.131103e-05 3.443731e-05 3.156378e-05 2.950385e-05
## Cumulative Proportion  9.997004e-01 9.997455e-01 9.997869e-01 9.998213e-01 9.998529e-01 9.998824e-01
##                              Comp61       Comp62       Comp63       Comp64       Comp65       Comp66
## Eigenvalues            1.298717e-07 1.099577e-07 1.054425e-07 8.319440e-08 7.878405e-08 6.718766e-08
## Proportion of Variance 2.659634e-05 2.251816e-05 2.159349e-05 1.703732e-05 1.613413e-05 1.375932e-05
## Cumulative Proportion  9.999090e-01 9.999315e-01 9.999531e-01 9.999701e-01 9.999862e-01 1.000000e+00</code></pre>
<div class="sourceCode" id="cb221"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb221-1"><a href="dgm---temporal.html#cb221-1" aria-hidden="true" tabindex="-1"></a><span class="co"># set plot parameters to plot by context</span></span>
<span id="cb221-2"><a href="dgm---temporal.html#cb221-2" aria-hidden="true" tabindex="-1"></a>pch.gps.context <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">15</span><span class="sc">:</span><span class="dv">18</span>)[<span class="fu">as.factor</span>(merged)]</span>
<span id="cb221-3"><a href="dgm---temporal.html#cb221-3" aria-hidden="true" tabindex="-1"></a>col.gps.context <span class="ot">&lt;-</span> pal[<span class="fu">as.factor</span>(merged)]</span>
<span id="cb221-4"><a href="dgm---temporal.html#cb221-4" aria-hidden="true" tabindex="-1"></a>col.hull.context <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;#C27D38&quot;</span>,<span class="st">&quot;#29211F&quot;</span>,<span class="st">&quot;#798E87&quot;</span>,<span class="st">&quot;#CCC591&quot;</span>)</span>
<span id="cb221-5"><a href="dgm---temporal.html#cb221-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb221-6"><a href="dgm---temporal.html#cb221-6" aria-hidden="true" tabindex="-1"></a><span class="do">## plot pca by context 2</span></span>
<span id="cb221-7"><a href="dgm---temporal.html#cb221-7" aria-hidden="true" tabindex="-1"></a>pc.plot <span class="ot">&lt;-</span> <span class="fu">plot</span>(pca, <span class="at">asp =</span> <span class="dv">1</span>,</span>
<span id="cb221-8"><a href="dgm---temporal.html#cb221-8" aria-hidden="true" tabindex="-1"></a>                <span class="at">pch =</span> pch.gps.context,</span>
<span id="cb221-9"><a href="dgm---temporal.html#cb221-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">col =</span> col.gps.context)</span>
<span id="cb221-10"><a href="dgm---temporal.html#cb221-10" aria-hidden="true" tabindex="-1"></a><span class="fu">shapeHulls</span>(pc.plot,</span>
<span id="cb221-11"><a href="dgm---temporal.html#cb221-11" aria-hidden="true" tabindex="-1"></a>           <span class="at">groups =</span> merged,</span>
<span id="cb221-12"><a href="dgm---temporal.html#cb221-12" aria-hidden="true" tabindex="-1"></a>           <span class="at">group.cols =</span> col.hull.context)</span></code></pre></div>
<p><img src="_main_files/figure-html/pca1.div-1.png" width="100%" /></p>
<div id="plot-pca-with-mean-shape-xy-maximaminima-1" class="section level3" number="7.4.1">
<h3><span class="header-section-number">7.4.1</span> Plot PCA with mean shape + x/y maxima/minima</h3>
<div class="sourceCode" id="cb222"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb222-1"><a href="dgm---temporal.html#cb222-1" aria-hidden="true" tabindex="-1"></a><span class="do">## plot PCA with mean shape (gray) and x/y maxima/minima (black)</span></span>
<span id="cb222-2"><a href="dgm---temporal.html#cb222-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/temporal-pca-warpref.png&#39;</span>)</span></code></pre></div>
<p><img src="images/temporal-pca-warpref.png" width="100%" /></p>
</div>
</div>
<div id="procrustes-anova-1" class="section level2" number="7.5">
<h2><span class="header-section-number">7.5</span> Procrustes ANOVA</h2>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Michael L. Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>. To assess whether shape changes with size (allometry), and differs by context (context), Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN1756" role="doc-biblioref">M. L. Collyer, Sekora, and Adams 2015</a>)</span>.</p>
<div class="sourceCode" id="cb223"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb223-1"><a href="dgm---temporal.html#cb223-1" aria-hidden="true" tabindex="-1"></a><span class="co"># MODEL: shape as a function of context + time</span></span>
<span id="cb223-2"><a href="dgm---temporal.html#cb223-2" aria-hidden="true" tabindex="-1"></a>fit.shapecontemp <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> merged,</span>
<span id="cb223-3"><a href="dgm---temporal.html#cb223-3" aria-hidden="true" tabindex="-1"></a>                             <span class="at">data =</span> gdf,</span>
<span id="cb223-4"><a href="dgm---temporal.html#cb223-4" aria-hidden="true" tabindex="-1"></a>                             <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb223-5"><a href="dgm---temporal.html#cb223-5" aria-hidden="true" tabindex="-1"></a>                             <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb223-6"><a href="dgm---temporal.html#cb223-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb223-7"><a href="dgm---temporal.html#cb223-7" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA: do gahagan biface shapes differ by context + time?</span></span>
<span id="cb223-8"><a href="dgm---temporal.html#cb223-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.shapecontemp)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df      SS        MS     Rsq      F      Z Pr(&gt;F)    
## merged     3 0.07959 0.0265306 0.24696 6.8871 3.4405  1e-04 ***
## Residuals 63 0.24269 0.0038522 0.75304                         
## Total     66 0.32228                                           
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ merged, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb225"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb225-1"><a href="dgm---temporal.html#cb225-1" aria-hidden="true" tabindex="-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb225-2"><a href="dgm---temporal.html#cb225-2" aria-hidden="true" tabindex="-1"></a>pairwise.shapecontemp <span class="ot">&lt;-</span> <span class="fu">pairwise</span>(fit.shapecontemp,</span>
<span id="cb225-3"><a href="dgm---temporal.html#cb225-3" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged)</span>
<span id="cb225-4"><a href="dgm---temporal.html#cb225-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pairwise.shapecontemp, </span>
<span id="cb225-5"><a href="dgm---temporal.html#cb225-5" aria-hidden="true" tabindex="-1"></a>        <span class="at">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb225-6"><a href="dgm---temporal.html#cb225-6" aria-hidden="true" tabindex="-1"></a>        <span class="at">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: initial_cache initial_individual subsequent_cache subsequent_individual 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                                                   d  UCL (95%)          Z Pr &gt; d
## initial_cache:initial_individual         0.05689191 0.06789423  1.3220896 0.1042
## initial_cache:subsequent_cache           0.01516509 0.03446703 -0.1269755 0.5384
## initial_cache:subsequent_individual      0.08386209 0.04232262  2.9240533 0.0002
## initial_individual:subsequent_cache      0.05951226 0.06704768  1.4408328 0.0857
## initial_individual:subsequent_individual 0.04466037 0.07023648  0.7341184 0.2490
## subsequent_cache:subsequent_individual   0.08100191 0.04105631  2.9828667 0.0001</code></pre>
<div class="sourceCode" id="cb227"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb227-1"><a href="dgm---temporal.html#cb227-1" aria-hidden="true" tabindex="-1"></a><span class="co"># MODEL: size as a function of context + time</span></span>
<span id="cb227-2"><a href="dgm---temporal.html#cb227-2" aria-hidden="true" tabindex="-1"></a>fit.sizecontemp <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(size <span class="sc">~</span> merged,</span>
<span id="cb227-3"><a href="dgm---temporal.html#cb227-3" aria-hidden="true" tabindex="-1"></a>                            <span class="at">data =</span> gdf,</span>
<span id="cb227-4"><a href="dgm---temporal.html#cb227-4" aria-hidden="true" tabindex="-1"></a>                            <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb227-5"><a href="dgm---temporal.html#cb227-5" aria-hidden="true" tabindex="-1"></a>                            <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb227-6"><a href="dgm---temporal.html#cb227-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb227-7"><a href="dgm---temporal.html#cb227-7" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA: do gahagan biface sizes differ by context + time?</span></span>
<span id="cb227-8"><a href="dgm---temporal.html#cb227-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sizecontemp)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df      SS     MS     Rsq      F      Z Pr(&gt;F)    
## merged     3  638396 212799 0.46852 18.512 5.0037  1e-04 ***
## Residuals 63  724190  11495 0.53148                         
## Total     66 1362586                                        
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = size ~ merged, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb229"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb229-1"><a href="dgm---temporal.html#cb229-1" aria-hidden="true" tabindex="-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb229-2"><a href="dgm---temporal.html#cb229-2" aria-hidden="true" tabindex="-1"></a>pairwise.sizecontemp <span class="ot">&lt;-</span> <span class="fu">pairwise</span>(fit.sizecontemp,</span>
<span id="cb229-3"><a href="dgm---temporal.html#cb229-3" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">groups =</span> qdata<span class="sc">$</span>merged)</span>
<span id="cb229-4"><a href="dgm---temporal.html#cb229-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pairwise.sizecontemp, </span>
<span id="cb229-5"><a href="dgm---temporal.html#cb229-5" aria-hidden="true" tabindex="-1"></a>        <span class="at">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb229-6"><a href="dgm---temporal.html#cb229-6" aria-hidden="true" tabindex="-1"></a>        <span class="at">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: initial_cache initial_individual subsequent_cache subsequent_individual 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                                                   d UCL (95%)          Z Pr &gt; d
## initial_cache:initial_individual         420.367335 153.20598  3.6341959 0.0001
## initial_cache:subsequent_cache            63.857495  77.82881  1.2287925 0.1126
## initial_cache:subsequent_individual        4.252466  96.92282 -1.5609099 0.9317
## initial_individual:subsequent_cache      356.509840 154.07650  3.2832616 0.0002
## initial_individual:subsequent_individual 416.114869 163.99286  3.4679877 0.0001
## subsequent_cache:subsequent_individual    59.605029  94.06328  0.8294343 0.2222</code></pre>
</div>
<div id="morphological-disparity-1" class="section level2" number="7.6">
<h2><span class="header-section-number">7.6</span> Morphological disparity</h2>
<div class="sourceCode" id="cb231"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb231-1"><a href="dgm---temporal.html#cb231-1" aria-hidden="true" tabindex="-1"></a><span class="co"># morphological disparity by shape</span></span>
<span id="cb231-2"><a href="dgm---temporal.html#cb231-2" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(fit.shapecontemp, </span>
<span id="cb231-3"><a href="dgm---temporal.html#cb231-3" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged, </span>
<span id="cb231-4"><a href="dgm---temporal.html#cb231-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb231-5"><a href="dgm---temporal.html#cb231-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb231-6"><a href="dgm---temporal.html#cb231-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.shapecontemp, groups = qdata$merged,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##         initial_cache    initial_individual      subsequent_cache subsequent_individual 
##           0.002595308           0.002590295           0.005141476           0.002601341 
## 
## 
## Pairwise absolute differences between variances
##                       initial_cache initial_individual subsequent_cache subsequent_individual
## initial_cache          0.000000e+00       5.012766e-06      0.002546168          6.033093e-06
## initial_individual     5.012766e-06       0.000000e+00      0.002551181          1.104586e-05
## subsequent_cache       2.546168e-03       2.551181e-03      0.000000000          2.540135e-03
## subsequent_individual  6.033093e-06       1.104586e-05      0.002540135          0.000000e+00
## 
## 
## P-Values
##                       initial_cache initial_individual subsequent_cache subsequent_individual
## initial_cache                1.0000             0.9974           0.0693                0.9974
## initial_individual           0.9974             1.0000           0.2657                0.9958
## subsequent_cache             0.0693             0.2657           1.0000                0.1263
## subsequent_individual        0.9974             0.9958           0.1263                1.0000</code></pre>
<div class="sourceCode" id="cb233"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb233-1"><a href="dgm---temporal.html#cb233-1" aria-hidden="true" tabindex="-1"></a><span class="co"># morphological disparity by size</span></span>
<span id="cb233-2"><a href="dgm---temporal.html#cb233-2" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(fit.sizecontemp, </span>
<span id="cb233-3"><a href="dgm---temporal.html#cb233-3" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged, </span>
<span id="cb233-4"><a href="dgm---temporal.html#cb233-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb233-5"><a href="dgm---temporal.html#cb233-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb233-6"><a href="dgm---temporal.html#cb233-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.sizecontemp, groups = qdata$merged,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##         initial_cache    initial_individual      subsequent_cache subsequent_individual 
##              4700.082             36093.906             11936.894             11493.575 
## 
## 
## Pairwise absolute differences between variances
##                       initial_cache initial_individual subsequent_cache subsequent_individual
## initial_cache                 0.000           31393.82        7236.8122             6793.4930
## initial_individual        31393.824               0.00       24157.0118            24600.3310
## subsequent_cache           7236.812           24157.01           0.0000              443.3192
## subsequent_individual      6793.493           24600.33         443.3192                0.0000
## 
## 
## P-Values
##                       initial_cache initial_individual subsequent_cache subsequent_individual
## initial_cache                1.0000             0.0059           0.1489                0.2731
## initial_individual           0.0059             1.0000           0.0229                0.0232
## subsequent_cache             0.1489             0.0229           1.0000                0.9435
## subsequent_individual        0.2731             0.0232           0.9435                1.0000</code></pre>
</div>
<div id="mean-shapes-1" class="section level2" number="7.7">
<h2><span class="header-section-number">7.7</span> Mean shapes</h2>
<div class="sourceCode" id="cb235"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb235-1"><a href="dgm---temporal.html#cb235-1" aria-hidden="true" tabindex="-1"></a><span class="co"># subset landmark coordinates to produce mean shapes for contexts</span></span>
<span id="cb235-2"><a href="dgm---temporal.html#cb235-2" aria-hidden="true" tabindex="-1"></a>new.coords <span class="ot">&lt;-</span> <span class="fu">coords.subset</span>(<span class="at">A =</span> Y.gpa<span class="sc">$</span>coords,</span>
<span id="cb235-3"><a href="dgm---temporal.html#cb235-3" aria-hidden="true" tabindex="-1"></a>                            <span class="at">group =</span> qdata<span class="sc">$</span>merged)</span>
<span id="cb235-4"><a href="dgm---temporal.html#cb235-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb235-5"><a href="dgm---temporal.html#cb235-5" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(new.coords)</span></code></pre></div>
<pre><code>## [1] &quot;initial_cache&quot;         &quot;initial_individual&quot;    &quot;subsequent_cache&quot;      &quot;subsequent_individual&quot;</code></pre>
<div class="sourceCode" id="cb237"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb237-1"><a href="dgm---temporal.html#cb237-1" aria-hidden="true" tabindex="-1"></a><span class="do">## plot shape means</span></span>
<span id="cb237-2"><a href="dgm---temporal.html#cb237-2" aria-hidden="true" tabindex="-1"></a>mean <span class="ot">&lt;-</span> <span class="fu">lapply</span>(new.coords, mshape)</span>
<span id="cb237-3"><a href="dgm---temporal.html#cb237-3" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>initial_cache)</span>
<span id="cb237-4"><a href="dgm---temporal.html#cb237-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>initial_individual)</span>
<span id="cb237-5"><a href="dgm---temporal.html#cb237-5" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>subsequent_cache)</span>
<span id="cb237-6"><a href="dgm---temporal.html#cb237-6" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mean<span class="sc">$</span>subsequent_individual)</span>
<span id="cb237-7"><a href="dgm---temporal.html#cb237-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-8"><a href="dgm---temporal.html#cb237-8" aria-hidden="true" tabindex="-1"></a><span class="co"># comparison plots</span></span>
<span id="cb237-9"><a href="dgm---temporal.html#cb237-9" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>initial_individual,</span>
<span id="cb237-10"><a href="dgm---temporal.html#cb237-10" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>initial_cache, </span>
<span id="cb237-11"><a href="dgm---temporal.html#cb237-11" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-12"><a href="dgm---temporal.html#cb237-12" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-13"><a href="dgm---temporal.html#cb237-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-14"><a href="dgm---temporal.html#cb237-14" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>initial_individual,</span>
<span id="cb237-15"><a href="dgm---temporal.html#cb237-15" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>subsequent_individual, </span>
<span id="cb237-16"><a href="dgm---temporal.html#cb237-16" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-17"><a href="dgm---temporal.html#cb237-17" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-18"><a href="dgm---temporal.html#cb237-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-19"><a href="dgm---temporal.html#cb237-19" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>initial_individual,</span>
<span id="cb237-20"><a href="dgm---temporal.html#cb237-20" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>subsequent_cache, </span>
<span id="cb237-21"><a href="dgm---temporal.html#cb237-21" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-22"><a href="dgm---temporal.html#cb237-22" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-23"><a href="dgm---temporal.html#cb237-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-24"><a href="dgm---temporal.html#cb237-24" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>initial_cache,</span>
<span id="cb237-25"><a href="dgm---temporal.html#cb237-25" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>subsequent_individual, </span>
<span id="cb237-26"><a href="dgm---temporal.html#cb237-26" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-27"><a href="dgm---temporal.html#cb237-27" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-28"><a href="dgm---temporal.html#cb237-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-29"><a href="dgm---temporal.html#cb237-29" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>subsequent_individual,</span>
<span id="cb237-30"><a href="dgm---temporal.html#cb237-30" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>initial_cache, </span>
<span id="cb237-31"><a href="dgm---temporal.html#cb237-31" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-32"><a href="dgm---temporal.html#cb237-32" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-33"><a href="dgm---temporal.html#cb237-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-34"><a href="dgm---temporal.html#cb237-34" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(mean<span class="sc">$</span>initial_cache,</span>
<span id="cb237-35"><a href="dgm---temporal.html#cb237-35" aria-hidden="true" tabindex="-1"></a>                mean<span class="sc">$</span>subsequent_cache, </span>
<span id="cb237-36"><a href="dgm---temporal.html#cb237-36" aria-hidden="true" tabindex="-1"></a>                <span class="at">method =</span> <span class="st">&quot;points&quot;</span>,</span>
<span id="cb237-37"><a href="dgm---temporal.html#cb237-37" aria-hidden="true" tabindex="-1"></a>                <span class="at">mag =</span> <span class="dv">1</span>)</span>
<span id="cb237-38"><a href="dgm---temporal.html#cb237-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb237-39"><a href="dgm---temporal.html#cb237-39" aria-hidden="true" tabindex="-1"></a><span class="do">## 3D plot - final figure</span></span>
<span id="cb237-40"><a href="dgm---temporal.html#cb237-40" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;images/comp.composite.jpg&#39;</span>)</span></code></pre></div>
<p><img src="images/comp.composite.jpg" width="100%" /></p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN1655" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package ’geomorph’: Geometric Morphometric Analyses of 2D/3D Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otárola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN1756" class="csl-entry">
Collyer, M. L., D. J. Sekora, and D. C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2021. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN11801" class="csl-entry">
———. 2018a. <span>“<span class="nocase">A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area</span>.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN3684" class="csl-entry">
Shafer, Harry J. 1973. <span>“<span class="nocase">Lithic Technology at the George C. Davis Site, Cherokee County, Texas</span>.”</span> Unpublished Ph.D. Dissertation, The University of Texas at Austin.
</div>
<div id="ref-RN20701" class="csl-entry">
———. 1974. <span>“<span class="nocase">Lithic Reduction Strategies at the George C. Davis Site</span>.”</span> <em>Louisiana Archaeology</em> 1: 66–74. <a href="https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf">https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf</a>.
</div>
<div id="ref-RN4924" class="csl-entry">
———. 2006. <span>“<span class="nocase">People of the Prairie: A Possible Connection to the Davis Site Caddo</span>.”</span> Texas Department of Transportation; Prewitt &amp; Associates, Inc.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, E., D. J. Gower, C. P. Klingenberg, and M. Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="dgm---individual-vs.-cache.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
